### PR TITLE
[4.4] Fix some leftovers

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | #33756
| License       | MIT
| Doc PR        | -

Found while running console commands with the Symfony Demo.

### CODE REVIEW

File: src/Symfony/Bundle/DebugBundle/Resources/config/services.xml [hunk 1]
```diff
@@ -96,7 +96,7 @@
             <argument type="collection">
                 <argument type="service" key="cli">
                     <service class="Symfony\Component\VarDumper\Command\Descriptor\CliDescriptor">
-                        <argument type="service" id="var_dumper.cli_dumper" />
+                        <argument type="service" id="var_dumper.contextualized_cli_dumper.inner" />
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/DebugBundle/Resources/config/services.xml]: /cc @ktherage FYI

File: src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php [hunk 2]
```diff
@@ -126,6 +126,10 @@ public function configure(Event $event = null)
                     $output = $output->getErrorOutput();
                 }
                 $this->exceptionHandler = function ($e) use ($app, $output) {
+                    if (!$e instanceof \Exception) {
+                        $e = new FatalThrowableError($e);
```
Comment (nicolas-grekas) [inline src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php]: /cc @fancyweb FYI

**Review by chalasr — APPROVED** [review]

File: src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php (hunk 2 above)
Comment (fancyweb) [reply src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php]: 👍 Might sounds useless but the function could be static + type hinted `\Throwable $e` to be consistent with what I did in https://github.com/symfony/symfony/pull/33038.

**Review by fancyweb — APPROVED** [review]

File: src/Symfony/Bundle/DebugBundle/Resources/config/services.xml (hunk 1 above)
Comment (ktherage) [reply src/Symfony/Bundle/DebugBundle/Resources/config/services.xml]: @nicolas-grekas Sorry, i've missed this one.

**Pull request merged**